tests/regression/80-termination/01-simple-loop-terminating.c

```diff
@@ -0,0 +1,15 @@
+// LOCAL_TERM PARAM: --set "ana.activated[+]" termination --enable warn.debug --set ana.activated[+] apron --enable ana.int.interval --set ana.apron.domain polyhedra
```

Comment:
Until commit 29568d2b8e3c29442152a223d4f1e5086231f1c5 the keywords `TERM` and `NONTERM` were already used by the old termination implementation. Since that commit is now complete, I think we can switch to `TERM` and `NONTERM`. I just needed to use some wording to separate our termination from the old termination.

If you know a better way than `TERM` and `NONTERM` it would probably be fine for me/us.

Reply:
`TERM` and `NONTERM` are fine with me.